engine/resources/nspawn.go

```diff
@@ -174,7 +172,6 @@ func (obj *NspawnRes) Watch() error {
 
 	defer close(busChan)
 	defer bus.Close()
-	defer bus.RemoveSignal(busChan)
```

Comment:
Let's make this clean, and keep this, but put it right after the add Signal command. You can add a comment about why it is or isn't absolutely needed if you want, but it's a nice thing to do for symmetry.